test/integrations/component.test.ts

```diff
@@ -124,6 +130,16 @@ const testRoute = async (route, tcData: TestCaseData) => {
   const outputResp = tcData.output.response || ({} as any);
   expect(response.status).toEqual(outputResp.status);
 
+  if (REPORT_COMPATIBLE_INTEGRATION.includes(tcData.name?.toLocaleLowerCase())) {
+    const bodyMatched = _.isEqual(response.body, outputResp.body);
+    const statusMatched = response.status === outputResp.status;
+    if (bodyMatched && statusMatched) {
+      generateTestReport(tcData, response.body, 'passed');
```

Comment:
is the report getting generated per test case?

Reply:
No it is appending in test you can check the function